js/lib/beautify-html.js

```diff
@@ -362,7 +377,7 @@
                     }
 
                     if (indent_handlebars && !tag_start_char) {
-                        if (content.length >= 2 && content[content.length - 1] === '{' && content[content.length - 2] == '{') {
+                        if (content.length >= 2 && content[content.length - 1] === '{' && content[content.length - 2] === '{') {
```

Comment:
I've made this change and the extra space clean up in the master branch now.  Thanks for bringing them up.